lib/addons/falco/index.ts

```diff
@@ -0,0 +1,80 @@
+import { HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+import { ClusterInfo } from '../../spi/types';
+import { HelmAddOn } from '../helm-addon/index';
+import { Construct } from "constructs";
+import { setPath } from "../../utils";
+import { Values } from "@aws-quickstart/eks-blueprints/dist/spi/types";
+
+
+
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface FalcoAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Version of the helm chart to deploy
+     */
+    version?: string,
+    /**
+     * Enable Kubernetes meta data collection via a connection to the Kubernetes API server
+     */
+    kubernetesSupportEnabled?: boolean,
+    /**
+     * Enable falcosidekick deployment
+     */
+    falcoSidekickEnabled?: string,
+    /**
+     * Create namespace for Falco
+     * @default falco
+     */
+    createNamespace?: boolean,
+}
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+ const defaultProps: HelmAddOnProps & FalcoAddOnProps = {
+    name: "blueprints-falco-addon",
+    namespace: "falco",
+    chart: "falco",
+    version: "1.17.4",
+    release: "falco",
+    repository:  "https://github.com/falcosecurity/charts",
+    createNamespace: true,
+    values: {
```

Comment:
We generally don't set defaults for the values. You have properties like kubernetesSupportEnabled, etc. at the root level. Defaults should follow the same logic.

Reply:
addressed